deltatech_product_update/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+Features:
+ -
```

Comment:
Nu ar trebui scris pe-aci ce face modulul?

Reply:
Am adaugat ce face modulul

Reply:
Da, asa am facut si noi la inceput, am pus o descriere scurta, iar dupa 3-4 ani a trebuit sa ne uitam in cod sa vedem ce face modulul.
Trebuie o descriere cat mai completa (inclusiv coloanele ce trebuie puse in excel)

Reply:
Am adaugat o descriere mai explicita